test/integration/image-component/basic/pages/blob.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from 'react'
```

Comment:
Let's move this test to `test/integration/image-component/default` and it can reuse the existing test image instead of copying a new one